packages/camera/camera_windows/lib/camera_windows.dart

```diff
@@ -50,14 +50,11 @@ class CameraWindows extends CameraPlatform {
   @override
   Future<List<CameraDescription>> availableCameras() async {
     try {
-      final List<String?> cameras = await _hostApi.getAvailableCameras();
+      final List<String> cameras = await _hostApi.getAvailableCameras();
 
-      return cameras.map((String? cameraName) {
+      return cameras.map((String cameraName) {
         return CameraDescription(
-          // This type is only nullable due to Pigeon limitations, see
-          // https://github.com/flutter/flutter/issues/97848. The native code
```

Comment:
This was fixed a while ago, I just missed updating this when I updated the Pigeon file.